src/software/control/Cargo.toml

```diff
@@ -20,12 +20,11 @@ doc = false
 log = "0.4"
 clap = { version = "2.33", default-features = false }
 lazy_static = "1.4"
-chrono = "0.4"
+chrono = "0.4.11"
 winit = "0.19"
-image = "0.23"
+image = "0.23.3"
 glium = "0.24"
-plotters = {version = "0.2", default_features = false, features = ["image_encoder", "chrono", "line_series", "palette", "ttf"]}
-piston_window = "0.105"
+plotters = {version = "^0.2.12", default_features = false, features = ["image_encoder", "chrono", "line_series", "palette", "ttf"]}
```

Comment:
Out of curiosity: what is the difference between `0.2` and `^0.2.12` here?

Reply:
None, it's the same, I left it after some debug, I'll clean it a bit